crates/common/tedge_config/src/tedge_config_cli/tedge_config.rs

```diff
@@ -730,6 +731,15 @@ define_tedge_config! {
         /// The filtering criterion that is used to filter packages list output by maintainer
         #[tedge_config(example = "thin-edge.io team.*")]
         maintainer: String,
+
+        dpk: {
+            options: {
+                /// Set the dpkg package manager configurable option
+                #[tedge_config(note = "If set to 'keepold', this keeps the old configuration files of the package that is being installed")]
```

Comment:
We should add to the note how these settings are mapped to the apt command, as people familiar with apt can easy see what the tedge option is mapped to.

```
The setting is used to control the dpkg options --force-confold and --force-confnew and are applied when installing apt packages.
```

Reply:
updated